python/ray/tests/test_state_api.py

```diff
@@ -1600,6 +1601,29 @@ def verify():
     print(list_actors())
 
 
+def test_list_actors_namespace(shutdown_only):
```

Comment:
Consider adding this check to existing list actor test? (to reduce ray.init load on tests)

Reply:
Good idea! I'm worried about cluttering the existing test with extra checks for a single field. I refactored the two unit tests to share a single class-scope fixture, so they both share a single `ray.init()` call. How does it look?